cypress/e2e/surveys.cy.ts

```diff
@@ -168,60 +167,6 @@ describe('Surveys', () => {
         cy.get('.Toastify__toast-body').contains('Survey deleted').should('be.visible')
     })
 
-    it('creates a new multiple choice survey with an open-ended choice', () => {
```

Comment:
don't remember the context for deleting this, but if its something to be fixed in the future, can we instead do `it.skip()` ?

Reply:
I've created an issue and saved the deleted test in a Gist attached: https://github.com/PostHog/posthog/issues/21877

But `it.skip()` is handy, will keep in mind for the next time 👍

Reply:
ah perf